Example/JustTweak/ConfigurationAccessor.swift

```diff
@@ -0,0 +1,64 @@
+//
+//  ConfigurationAccessor.swift
+//  JustTweak_Example
+//
+//  Created by Alberto De Bortoli on 02/11/2019.
+//  Copyright © 2019 Just Eat. All rights reserved.
+//
+
+import Foundation
+import JustTweak
+
+class ConfigurationAccessor {
+    
+    lazy var configurationsCoordinator: TweaksConfigurationsCoordinator = {
```

Comment:
reason why this is not injected will be revealed with the next PR introducing property wrappers